yoti-sdk-sandbox/src/main/java/com/yoti/api/client/sandbox/docs/request/check/SandboxCheckBuilder.java

```diff
@@ -0,0 +1,37 @@
+package com.yoti.api.client.sandbox.docs.request.check;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.yoti.api.client.sandbox.docs.request.check.report.SandboxBreakdown;
+import com.yoti.api.client.sandbox.docs.request.check.report.SandboxRecommendation;
+
+abstract class SandboxCheckBuilder<T extends SandboxCheckBuilder<T>> {
+
+    protected SandboxRecommendation recommendation;
+    protected List<SandboxBreakdown> breakdown;
+
+    public T withRecommendation(SandboxRecommendation recommendation) {
+        this.recommendation = recommendation;
+        return self();
+    }
+
+    public T withBreakdown(SandboxBreakdown breakdown) {
+        if (this.breakdown == null) {
+            this.breakdown = new ArrayList<>();
+        }
```

Comment:
@MrBurtyyy as discussed, we can default this to an empty list

@echarrod .NET validates this is not an empty list, which might not be necessary (if we feel the SDK should validate this, we need to update PHP/Node)